src/main/resources/api.yaml

```diff
@@ -1438,6 +1438,9 @@ definitions:
         type: array
         items:
           type: string
+      selfRegistrationAllowed:
+        type: boolean
+        description: Self Registration Allowed
```

Comment:
Could we add a bit more info: "Flag indicating where Self-service registration is enabled for this service. If omitted, defaults to false."